iothub_client/tests/common_longhaul/iothub_client_common_longhaul.c

```diff
@@ -1396,7 +1396,26 @@ static int invoke_device_method(const void* context)
                 MAX_DEVICE_METHOD_TRAVEL_TIME_SECS,
                 &responseStatus, &responsePayload, &responseSize)) != IOTHUB_DEVICE_METHOD_OK)
             {
-                LogError("Failed invoking device method");
+                if (responseStatus == 404)
```

Comment:
If we are going to retry instead of reconciling the results with reconnection data, we should put this in a loop of 2 iterations instead of duplicating the code I think. What do you say?

Reply:
made retry a loop